test/sagemaker_tests/tensorflow/inference/test/integration/local/test_tfs_batching.py

```diff
@@ -31,6 +31,7 @@ def volume():
         subprocess.check_call('docker volume rm batching_model_volume'.split())
 
 
+@pytest.mark.integration("tfs batching")
```

Comment:
double check if this is the right integration

Reply:
Seems to be the only occurrence of a "batching_parameters" test. So not sure what to double check against.

nit: It would be better if it was `tfs_batching` rather than `tfs batching`, but this is not an approval blocker.